packages/wonder-blocks-tokens/src/theme/semantic/semantic-color.ts

```diff
@@ -64,11 +132,15 @@ const surface = {
     overlay: color.offBlack64,
 };
 
+/**
+ * TODO(WB-1941): Remove text once we have migrated to the new core.foreground
+ * tokens.
+ */
 const text = {
-    primary: color.offBlack,
-    secondary: color.fadedOffBlack72,
-    disabled: color.fadedOffBlack32,
-    inverse: color.white,
+    primary: core.foreground.neutral.strong,
```

Comment:
I found the previous semantic text tokens intuitive to use since `text.primary` should be used for most text, and `text.secondary` for subtle text!

Since we will be getting rid of these text tokens, consumers will be using the core tokens directly for text (if I'm understanding correctly!). Based on the semantic naming, I thought that `core.foreground.neutral.default` would be the primary text color that's most used and `core.foreground.neutral.subtle` would be the secondary text color. And `core.foreground.neutral.strong` would be for emphasized text or something like that! Here, I see that `strong` is the default primary text.

This might just be my own assumptions though, but was curious what you (and others) think about this!

Reply:
That said, I'm going to keep these tokens for now, but we'll see how these can be used in the system and how/if we use them in product, and we can get back to implement any required changes based on usage.